spec/integration/datacenter_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Datacenter' do
```

Comment:
These tests look like the default `find_by_id_and_name` specs. Can't we use that default things too?